state/txindex/kv/utils.go

```diff
@@ -106,3 +107,23 @@ func checkHeightConditions(heightInfo HeightInfo, keyHeight int64) (bool, error)
 	}
 	return true, nil
 }
+
+// GetKeys is used for testing purposes only
+func GetKeys(indexer *TxIndex) [][]byte {
```

Comment:
Can we make this not public, or be in `internal/indexer_utils`?

Reply:
Not really, since it accesses `indexer.store`...